source/dofs/dof_handler_policy.cc

```diff
@@ -3182,14 +3182,26 @@ namespace internal
 
 
 
-        template <int dim, int spacedim>
+        template <int spacedim>
         void
-        set_dofindices_recursively (
-          const parallel::distributed::Triangulation<dim,spacedim> &tria,
-          const typename dealii::internal::p4est::types<dim>::quadrant &p4est_cell,
-          const typename DoFHandler<dim,spacedim>::level_cell_iterator &dealii_cell,
-          const typename dealii::internal::p4est::types<dim>::quadrant &quadrant,
-          dealii::types::global_dof_index *dofs)
+        communicate_mg_ghost_cells(const typename parallel::distributed::Triangulation<1,spacedim> &,
+                                   hp::DoFHandler<1,spacedim> &,
+                                   const std::vector<dealii::types::global_dof_index> &,
+                                   const std::vector<dealii::types::global_dof_index> &)
+        {
+          Assert (false, ExcNotImplemented());
+        }
+
+
+
+
```

Comment:
Since you update the number of blank lines in this file, this should also be three blank lines.